Jenkinsfile

```diff
@@ -6,11 +6,12 @@ pipeline {
         buildDiscarder(logRotator(numToKeepStr:'25'))
     }
     triggers {
-        cron('H H(20-23) * * *')
+        cron('H H(17-19) * * *')
     }
     environment {
         DOCS = 'distribution/docs'
         ITESTS= 'distribution/test/itests/test-itests-ddf'
+        JDK_LOC = 'C:/Program Files/Java/jdk1.8.0_131'
```

Comment:
I figured it was marginally volatile so I chose to place it here even though it is only used once below. I don't imagine we would update it frequently to be honest.